megamek/src/megamek/common/MechView.java

```diff
@@ -252,6 +252,8 @@ public MechView(final Entity entity, final boolean showDetail, final boolean use
                     Messages.getString("MechView.Source")));
             if (!entity.getSource().isEmpty()) {
                 sHead.add(new LabeledElement("", entity.getSource()));
+            } else {
+                sHead.add(new LabeledElement("", "(unkown)"));
```

Comment:
It should be "unknown", not "unkown"

Reply:
I must have been really tired and impatient...